scripts/tests/lib.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -euo pipefail
```

Comment:
This forces the top-level script to have `set -euo pipefail` and means you can't source `lib.sh` from an interactive session. The first error will exit the whole Bash session

Reply:
Ah, I think we've assumed these scripts are just for CI.

Are they intended to be run locally?

Reply:
This doesn't have to go in; I don't feel strongly either way. 

A case could be made against using `set ...` in lib scripts anyhow